conf/application.conf

```diff
@@ -27,17 +42,16 @@ microservice {
 play.i18n.langCookieHttpOnly: "true"
 
 # Change this value to true to enable Welsh translations to be loaded from messages.cy, and to display the language toggle
-features.welsh-language-support = false
+features.welsh-language-support = true
 
 # Replace play.i18n.langs with the commented out line below when your service has been fully translated into Welsh
 # to enable Welsh translations for all content, including the standard headers and footers.
 # play.i18n.langs = ["en", "cy"]
-play.i18n.langs = ["en"]
+play.i18n.langs = ["en", "cy"]
```

Comment:
It feels messy to have both the commented and uncommented. What do you think of deleting the commented line?